### PR TITLE
Update arte.xml

### PR DESCRIPTION
**Pull request by I-have-50-open-PRs-to-HTTPS-EVERYWHERE**

### CODE REVIEW

File: src/chrome/content/rules/arte.xml [hunk 1]
```diff
@@ -64,19 +57,25 @@ Fetch error: http://info.arte.tv/ => https://info.arte.tv/: Too many redirects w
 
 	wrong content:
 		- forums.arteradio.com
+
+	Redirect to HTTP:
+		- info.arte.tv
+
+	Too many redirects:
```
Comment (koops76) [inline src/chrome/content/rules/arte.xml]: Aren't `Redirect to HTTP` and `Too many redirects` the same?

File: src/chrome/content/rules/arte.xml [hunk 2]
```diff
@@ -64,19 +57,25 @@ Fetch error: http://info.arte.tv/ => https://info.arte.tv/: Too many redirects w
...
 	<target host="club.arte.tv" />
 	<target host="concert.arte.tv" />
-	<target host="future.arte.tv" />
-	<target host="info.arte.tv" />
 	<target host="arteradio.com" />
 	<target host="www.arteradio.com" />
 
+	<securecookie host=".+" name=".+" />
+
 	<rule from="^http:"
```
Comment (koops76) [inline src/chrome/content/rules/arte.xml]: You may remove a line break here, but it's optional.

File: src/chrome/content/rules/arte.xml [hunk 3]
```diff
@@ -64,19 +57,25 @@ Fetch error: http://info.arte.tv/ => https://info.arte.tv/: Too many redirects w
...
+<ruleset name="arte (partial)" >
 	<target host="arte.tv" />
 	<target host="www.arte.tv" />
 	<target host="boutique.arte.tv" />
-	<target host="cinema.arte.tv" />
 	<target host="club.arte.tv" />
 	<target host="concert.arte.tv" />
-	<target host="future.arte.tv" />
-	<target host="info.arte.tv" />
 	<target host="arteradio.com" />
```
Comment (koops76) [inline src/chrome/content/rules/arte.xml]: Maybe add a line break between different domains?

File: src/chrome/content/rules/arte.xml [hunk 4]
```diff
@@ -1,12 +1,5 @@
 
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://arte.tv/ => https://arte.tv/: Too many redirects while fetching 'https://arte.tv/'
-Fetch error: http://www.arte.tv/ => https://www.arte.tv/: Too many redirects while fetching 'https://www.arte.tv/'
-Fetch error: http://cinema.arte.tv/ => https://cinema.arte.tv/: Too many redirects while fetching 'https://cinema.arte.tv/'
-Fetch error: http://future.arte.tv/ => https://future.arte.tv/: Too many redirects while fetching 'https://future.arte.tv/'
-Fetch error: http://info.arte.tv/ => https://info.arte.tv/: Too many redirects while fetching 'https://info.arte.tv/'
-
 	Mismatch:
```
Comment (koops76) [inline src/chrome/content/rules/arte.xml]: `Mismatched`?

File: src/chrome/content/rules/arte.xml [hunk 5]
```diff
@@ -64,19 +57,25 @@ Fetch error: http://info.arte.tv/ => https://info.arte.tv/: Too many redirects w
 
 	wrong content:
```
Comment (koops76) [inline src/chrome/content/rules/arte.xml]: Capitalization.

File: src/chrome/content/rules/arte.xml (hunk 1 above)
Comment (I-have-50-open-PRs-to-HTTPS-EVERYWHERE) [inline src/chrome/content/rules/arte.xml]: @koops76 They have different behavior here (one redirects to HTTP other to HTTPS after a while) but i'm ok with what you say

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@koops76 Added.

File: src/chrome/content/rules/arte.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/arte.xml]: @I-have-50-open-PRs-to-HTTPS-EVERYWHERE Do you mean it sometimes redirects to HTTP, but sometimes it doesn't?

File: src/chrome/content/rules/arte.xml [hunk 6]
```diff
@@ -62,21 +55,25 @@ Fetch error: http://info.arte.tv/ => https://info.arte.tv/: Too many redirects w
...
 	<target host="club.arte.tv" />
 	<target host="concert.arte.tv" />
-	<target host="future.arte.tv" />
-	<target host="info.arte.tv" />
+
 	<target host="arteradio.com" />
 	<target host="www.arteradio.com" />
 
+	<securecookie host=".+" name=".+" />
 	<rule from="^http:"
```
Comment (koops76) [inline src/chrome/content/rules/arte.xml]: No, add the line break between `securecookie` and `rule` tags and remove the one inside the `rule` tag.

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@koops76 added.

**Comment by koops76** [comment]
@J0WI Review this please.

**Pull request merged**